### PR TITLE
Renders 404 if table accessed is raster kind

### LINKED ISSUE #6191: Raster tables can be accessed by hardcoding the URL and are available in write-mode
While debugging: https://github.com/CartoDB/cartodb/issues/1232

I noticed that although raster tables are listed as "non-accessible" in the dashboard, you can just make up the URL to see them.

Once the table view is open, these tables are not set as in read-only mode and their fields are editable or even you can force a georeferenciation process on them.

### PR DESCRIPTION
**Pull request by gfiorav**
Fixes #6191

### CODE REVIEW

**Comment by gfiorav** [comment]
After speaking with @xavijam , we've split the front end and back end work. The front end ticket is #6377 

Once CI passes, I'll ask for CR.

**Comment by gfiorav** [comment]
Hey @juanignaciosl could you review this?

The front end work will be completed in the future. In the mean time this is the back end part.

A bit hacky maybe, totally up for discussing a better solution.

File: app/controllers/admin/visualizations_controller.rb [hunk 1]
```diff
@@ -618,6 +618,7 @@ def get_visualization_and_table(table_id, schema, filter)
                                                       }
                                                     .first
 
+    render_pretty_404 if !visualization.nil? && visualization.kind == Visualization::Member::KIND_RASTER
```
Comment (juanignaciosl) [inline app/controllers/admin/visualizations_controller.rb]: - Move this line under the next one and you can safely remove the `.nil?` check.
- Replace `Visualization::Member::KIND_RASTER` with `CartoDB::Visualization::Member::KIND_RASTER`, both are almost equivalent but first one can be problematic in some corner cases because of class loading order (tests and so on).

I'd also like the error page to display an specific message about "You can't open a raster visualization in the editor" or something similar, but that's a frontend decision for sure. // @xavijam

Comment (gfiorav) [inline app/controllers/admin/visualizations_controller.rb]: Ok @juanignaciosl ! Thanks for these comments. I'll refactor to them.

**Comment by gfiorav** [comment]
See if it looks good now, @juanignaciosl 

Thanks!

**Comment by juanignaciosl** [comment]
:+1:
